be/src/exprs/expr.h

```diff
@@ -139,6 +139,8 @@ class Expr {
     // Returns the number of slots added to the vector
     virtual int get_slot_ids(std::vector<SlotId>* slot_ids) const;
 
+    virtual int get_subfields(std::vector<std::vector<std::string>>* subfields) const;
```

Comment:
Add some comments to explain it.

Reply:
should not subfields just be vector<string> ?

Reply:
subfield is a vector which is the path, example col.a.b.c for leaf subfield c the path is vector{a,b,c}; 
but an expr may contain more than one subfield, so it needs vector<vector<>>